x-pack/plugin/profiling/src/main/java/org/elasticsearch/xpack/profiling/CO2Calculator.java

```diff
@@ -26,21 +26,26 @@ final class CO2Calculator {
     private final double customCO2PerKWH;
     private final double customDatacenterPUE;
     private final double customKilowattsPerCore;
+    private final double customKilowattsPerCoreARM64;
 
     CO2Calculator(
         InstanceTypeService instanceTypeService,
         Map<String, HostMetadata> hostMetadata,
         double samplingDurationInSeconds,
         Double customCO2PerKWH,
         Double customDatacenterPUE,
-        Double customPerCoreWatt
+        Double customPerCoreWatt,
+        Double customPerCoreWattARM64
     ) {
         this.instanceTypeService = instanceTypeService;
         this.hostMetadata = hostMetadata;
         this.samplingDurationInSeconds = samplingDurationInSeconds > 0 ? samplingDurationInSeconds : 1.0d; // avoid division by zero
         this.customCO2PerKWH = customCO2PerKWH == null ? DEFAULT_CO2_TONS_PER_KWH : customCO2PerKWH;
         this.customDatacenterPUE = customDatacenterPUE == null ? DEFAULT_DATACENTER_PUE : customDatacenterPUE;
         this.customKilowattsPerCore = customPerCoreWatt == null ? DEFAULT_KILOWATTS_PER_CORE : customPerCoreWatt / 1000.0d;
```

Comment:
Should we rename this field to `customKilowattsPerCoreX86`? I suppose that we will eventually get rid of the special treatment for ARM. In that case we can also keep the field name as is; up to you.

Reply:
So the `customKilowattsPerCore` is not only for x86, it is a default, so I decided against adding `X86` as a suffix. This can change in the future, but I don't expect changes soon (e.g. riscv isn't an option for aws, gcp or azure, AFAIK).